bootstrap.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python
+# coding=utf-8
+import argparse
+import os
+import subprocess
+from string import Template
+
+from blessings import Terminal
+
+
+DEFAULT_DIRECTORY_NAME = 'mozilla-payments'
+GIT_REPOS = [
+    {'org': 'Kinto', 'name': 'kinto'},
+    {'org': 'mozilla', 'name': 'payments-example'},
+    {'org': 'mozilla', 'name': 'payments-service'},
+    {'org': 'mozilla', 'name': 'payments-ui'},
+    {'org': 'mozilla', 'name': 'solitude'},
+    {'org': 'mozilla', 'name': 'solitude-auth'},
+]
+PAYMENT_ENV_DIR = os.path.abspath(os.path.dirname(__file__))
+LOCAL_ENV = Template((os.fdopen(os.open(
+    os.path.join(PAYMENT_ENV_DIR, 'env', 'local.env.dist'),
+    os.O_RDWR), "r+").read()))
+SEPARATOR = "\n###############################################################################\n💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰\n###############################################################################"  # nopep8
```

Comment:
Oops, I should do that instead! Thanks ^_^


- tofumatt

On Wed, Sep 23, 2015 at 6:25 PM, Fred Wenzel <notifications@github.com>
wrote:

>> +
>> +
>> +DEFAULT_DIRECTORY_NAME = 'mozilla-payments'
>> +GIT_REPOS = [
>> +    {'org': 'Kinto', 'name': 'kinto'},
>> +    {'org': 'mozilla', 'name': 'payments-example'},
>> +    {'org': 'mozilla', 'name': 'payments-service'},
>> +    {'org': 'mozilla', 'name': 'payments-ui'},
>> +    {'org': 'mozilla', 'name': 'solitude'},
>> +    {'org': 'mozilla', 'name': 'solitude-auth'},
>> +]
>> +PAYMENT_ENV_DIR = os.path.abspath(os.path.dirname(__file__))
>> +LOCAL_ENV = Template((os.fdopen(os.open(
>> +    os.path.join(PAYMENT_ENV_DIR, 'env', 'local.env.dist'),
>> +    os.O_RDWR), "r+").read()))
>> +SEPARATOR = "\n###############################################################################\n💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰 💰\n###############################################################################"  # nopep8
> you can repeat a string with `'#' * 78`, btw
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/mozilla/payments-env/pull/55/files#r40231583